### PR TITLE
Include AbtionGPT as an iframe in a new page

### PR DESCRIPTION
**Pull request by davideluque**

### CODE REVIEW

**Comment by davideluque** [comment]
Screenshot 2023-05-23 at 17 17 42

Screenshot 2023-05-23 at 17 19 25

**Comment by hcarreras** [comment]
Super cool.

I'd like the main page to have the iframe. Then, we can move the philosophy and contribution to another page. 
I'll make the changes to the PR myself. 
Commit coming in a few minutes.

File: artificial_intelligence/abtion_gpt.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# AbtionGPT
+
+<iframe src="http://localhost:3001" style="width:100%;height:100vh;"> </iframe>
```
[the remark this answers is not in the document]
Comment (davideluque) [reply artificial_intelligence/abtion_gpt.md:3]: Ow... I thought i pushed the right one.

[the remark this answers is not in the document]
Comment (hcarreras) [reply artificial_intelligence/abtion_gpt.md]: Cool. I've moved it to a var

**Pull request merged**